apps/frontend/src/libs/components/auth-wrapper.tsx/auth-wrapper.tsx

```diff
@@ -0,0 +1,24 @@
+// import { QuizForm } from "~/pages/quiz/libs/components/quiz-form/quiz-form.jsx";
```

Comment:
```suggestion
```

Reply:
After change the prop into root. the UI look like this

![Screenshot from 2024-08-26 23-08-54](https://github.com/user-attachments/assets/a6e80dc2-bf60-49a1-9721-782a9a74e6be)